src/modules/esl-modal/core/esl-modal.less

```diff
@@ -0,0 +1,64 @@
+esl-modal {
+  display: none;
+}
+
+.esl-modal {
+  position: fixed;
+  z-index: 1000000;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: var(--100vh, 100vh);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow-y: scroll;
+
+  &:not(.open) {
+    display: none;
+  }
+
+  &::before, &::after {
+    content: '';
+    flex: 1 0 30px;
+  }
+
+  .esl-modal-container {
+    flex: 0 0 auto;
+    position: relative;
+    padding: 50px;
+    max-width: 100%;
+    width: 80%;
+    background-color: #fff;
+    overflow: auto;
+  }
+
+  .close-btn {
+    position: absolute;
+    right: 15px;
+    top: 15px;
```

Comment:
Can we use inset CSS properties to resolve cases with different writing modes, directionality, and text orientation?

Reply:
It's good idea!)